src/node/progress_tracker_types.h

```diff
@@ -301,4 +301,20 @@ namespace ccf
 
     return 2 * f + 1;
   }
+
+  template <typename T>
```

Comment:
Add a comment explaining endorsements can be view changes, or signatures over root/state, or nonces...